python/cuml/cuml/svm/svm_base.pyx

```diff
@@ -295,6 +422,7 @@ class SVMBase(UniversalBase,
                 pass
 
         self._model = None
+        self._freeSvmBuffers = False
```

Comment:
Should we set `self._freeSvmBuffers` to True in `_get_svm_model`?

Reply:
I don't think so? It's set to True in `_unpack_model`. IIUC since the vectors here aren't owned by the model then the model shouldn't be responsible for deallocating them. In the case of `fit` though the output vectors all are owned by the wrapped model and would need to be deallocated later.

The memory management here is a bit of a mess though and not following cython best practices. I'd like to refactor this, but in a follow-up.